### ISSUE TITLE
kWh in 'Drive Details' doesn't match 'Drives' dashboard

### ISSUE THREAD

**Issue by bpankz** [adriankumpf/teslamate#586]
Drive dashboard shows the correct kWh usage number but Drive Details dashboard shows a higher number. The sql is converting km to miles by multiplying by 1.6 but the number in the database seems to be stored as miles already.

**Screenshots**
Drives
If applicable, add screenshots to help explain the problem.
![image](https://user-images.githubusercontent.com/54423181/79160976-b3a2f380-7da8-11ea-9dcb-4573a215b333.png)

Drive Details
should be 16.9kwh
![image](https://user-images.githubusercontent.com/54423181/79161127-f5cc3500-7da8-11ea-9d84-b29439b1a821.png)

should be 23.6kwh
![image](https://user-images.githubusercontent.com/54423181/79161218-16948a80-7da9-11ea-992e-b135df783c2f.png)

changing this
```SELECT
	(NULLIF(GREATEST(start_[[preferred_range]]_range_km - end_[[preferred_range]]_range_km, 0), 0) * car.efficiency) * 
	  CASE WHEN '$length_unit' = 'km' THEN 1
		  	 WHEN '$length_unit' = 'mi' THEN 1.60934
	  END
FROM
	drives d
JOIN cars car ON car.id = car_id
WHERE
	d.id = $drive_id;
```

to this displays as expected in the drive details dashboard

```SELECT
	(NULLIF(GREATEST(start_[[preferred_range]]_range_km - end_[[preferred_range]]_range_km, 0), 0) * car.efficiency)
FROM
	drives d
JOIN cars car ON car.id = car_id
WHERE
	d.id = $drive_id;
```

**Comment by adriankumpf** [comment]
This was fixed in b67b6e6f43145def4443e7d71c3d32d99aa304a5 (v1.16.0). It seems the dashboard(s) need to be re-imported?

**Comment by bpankz** [comment]
still had a drive-details outside of the internal folder that kepy importing and was old. removed and restored, all works now
